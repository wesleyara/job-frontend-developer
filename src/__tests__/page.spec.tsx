import { beforeEach, describe, expect, it } from "vitest";

import Page from "../app/page";

describe("Page unit tests", () => {
  let sut: JSX.Element;

  beforeEach(() => {
    sut = <Page />;
  });

  it("should mounted the page", () => {
    expect(sut).toBeDefined();
  });
});
