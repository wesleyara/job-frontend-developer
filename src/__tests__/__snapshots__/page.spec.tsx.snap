// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Page unit tests > should render correctly 1`] = `
<div>
  <main>
    <h1>
      Welcome to the Search Band App
    </h1>
    <img
      alt="Next.js logo"
      data-nimg="1"
      decoding="async"
      height="200"
      loading="lazy"
      src="/next.svg"
      style="color: transparent;"
      width="200"
    />
  </main>
</div>
`;
